src/main/java/ch/njol/skript/expressions/ExprDestroyableKeys.java

```diff
@@ -0,0 +1,140 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import com.destroystokyo.paper.Namespaced;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
+
+@Name("Destroyable Keys Of Item")
+@Description("Get or modify the destroyable keys of an item.")
+@Examples({"add dirt to destroyable keys of player's tool",
+	"add stone and diamond ore to destroy blocks of {_item}",
+	"clear destroyable items of {_item}",
+	"remove sand from destroy keys of {_item}"})
+@Since("x.x.x") // 2.9.0?
+public class ExprDestroyableKeys extends PropertyExpression<ItemType, NamespacedKey> {
```

Comment:
Skript doesn't have a namespacedkey type, returning it won't be useful! This should return item types.